docs/sources/panels/panel-editor.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Panel editor"
+type = "docs"
+[menu.docs]
+identifier = "panel-editor"
+weight = 200
++++
+
+# Panel editor
+
+This page describes the parts of the Grafana panel editor and links to where you can find more information.
+
+{{< docs-imagebox img="/img/docs/panel-editor/panel-editor-7-0.png" class="docs-image--no-shadow" max-width="1500px" >}}
```

Comment:
https://github.com/grafana/website/blob/60b482af9068d160eecc9d38d58fba8a1842eb78/static/static/img/docs/panel-editor/panel-editor-7-0.png 

image does not seem ideal, very high res with lots of empty space and with side panel collapsed. would use something like: 
<img width="1614" alt="Screen Shot 2020-05-14 at 19 46 45" src="https://user-images.githubusercontent.com/10999/82069228-4a7a0d00-96d3-11ea-8623-c5ec9fadd17c.png">

Reply:
The point of the image is to show all the parts of the UI. This one does not do that. 

I resized my image and trimmed the empty space.

Reply:
sorry wong image

Reply:
I meant this

<img width="1680" alt="Screen Shot 2020-05-15 at 17 40 59" src="https://user-images.githubusercontent.com/10999/82072306-cfffbc00-96d7-11ea-92fb-852f770eb301.png">

Reply:
Ooh, pretty! Yes, I'll add that one.